### PR TITLE
Fix #1423 : Added 'How to use vs2019'

### LINKED ISSUE #1423: Add 'How to use vs2019' to the docs.
Add 'How to use vs2019' to the docs.

### PR DESCRIPTION
**Pull request by ForgottenProgramme**
PR Checklist:

- [X] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [X] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [X] put any other relevant information below

Described the changes required in the conda_build_config.yaml file and the conda-forge.yml file .

Fixes #1423

### CODE REVIEW

File: src/maintainer/knowledge_base.rst [hunk 1]
```diff
@@ -189,8 +189,35 @@ To skip building with a particular ``vc`` version, add a skip statement.
...
+
+.. code-block:: yaml
+
+    c_compiler:                    
+    - vs2019                       
+    cxx_compiler:                  
+    - vs2019                       
 
 
+.. code-block::
```
[the remark this answers is not in the document]
Comment (ForgottenProgramme) [reply src/maintainer/knowledge_base.rst]: Changed! :)

File: src/maintainer/knowledge_base.rst [hunk 2]
```diff
@@ -189,8 +189,36 @@ To skip building with a particular ``vc`` version, add a skip statement.
...
+
+.. code-block:: yaml
 
+    azure:
+      settings_win:
+          pool:
+              vmImage: windows-2019
+     
+     
+.. code-block::
```
Comment (wolfv) [inline src/maintainer/knowledge_base.rst:218]: this should also not be needed, right?

**Comment by wolfv** [comment]
We should also add a note that after these changes one needs to "rerender". maybe link to another part of the docs where that is described.

**Comment by ForgottenProgramme** [comment]
oh, yes, Thankyou. 
Added `conda smithy rerender` .

**Comment by wolfv** [comment]
thanks!

**Pull request merged**